<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeUI</class>
 <widget class="QDialog" name="WelcomeUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hargoile</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="loginFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhEmailCharactersOnly|Qt::ImhLowercaseOnly</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoPredictiveText</set>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="menuButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
